src/main/java/in/projecteka/consentmanager/clients/ClientError.java

```diff
@@ -308,4 +310,20 @@ public static ClientError unprocessableEntity() {
 
     }
 
+    public static ClientError invalidRecoveryRequest() {
+        return new ClientError(BAD_REQUEST,
+                new ErrorRepresentation(new Error(INVALID_RECOVERY_REQUEST, "Invalid CM Id recovery request")));
+    }
+
+    public static ClientError noPatientFound() {
+        return new ClientError(NOT_FOUND,
+                new ErrorRepresentation(new Error(NO_PATIENT_FOUND, "No patient matching the records")));
+    }
+
+    public static ClientError multiplePatientsFound() {
```

Comment:
I think we should not reveal any additional info regarding whether multiple patients were identified in the process of recovery. Taking other system references for recovery, none says so in the recovery process. I think we should throw a generic message saying - can not identify patient, insufficient info and also not send out a specific MULTIPLE_PATIENTS_FOUND error. Check with Akshatha.

Reply:
sending NO_PATIENT_FOUND for both zero or multiple patients found cases.